scripts/azure-linux_mac.yml

```diff
@@ -0,0 +1,190 @@
+steps:
+- bash: |
+    set -e pipefail
+    # Install doxygen *before* running cmake
+    sudo apt-get install doxygen
+  condition: eq(variables['Agent.OS'], 'Linux')
+  displayName: 'Install doxygen (linux only)'
+
+
+- bash: |
+    set -e pipefail
+    # Install clang-format (v5.0)
+    sudo scripts/install-clangformat.sh
+    src=$BUILD_REPOSITORY_LOCALPATH
+    cd $src
+    $src/scripts/run-clang-format.sh $src clang-format-5.0 0 \
+      $(find $src/tiledb $src/test/src $src/examples $src/tools \
+       -name "*.cc" -or -name "*.c" -or -name "*.h")
+  condition: eq(variables['Agent.OS'], 'Linux')
+  displayName: 'Check formatting (linux only)'
+
+
+- bash: |
+    # azure bash does not treat intermediate failure as error
+    # https://github.com/Microsoft/azure-pipelines-yaml/issues/135
+    set -e pipefail
+
+    git clone https://github.com/TileDB-Inc/TileDB-Unit-Test-Arrays.git test/inputs/arrays/read_compatibility_test
+    #   displayName: 'Clone Unit-Test-Arrays'
+
+    # - bash: |
+    # Start HDFS server if enabled
+    if [[ "$TILEDB_HDFS" == "ON" ]]; then
+      # - ssh to localhost is required for HDFS launch...
+      # - /home/vsts has permissions g+w and is owned by user 'docker'
+      #   for VSTS purposes, so disable ssh strictness
+      sudo sed -i "s/StrictModes\ yes/StrictModes\ no/g" /etc/ssh/sshd_config
+
+      source scripts/install-hadoop.sh
+      source scripts/run-hadoop.sh
+    fi
+
+    # Start minio server if S3 is enabled
+    if [[ "$TILEDB_S3" == "ON" ]]; then
+      source scripts/install-minio.sh;
+      source scripts/run-minio.sh;
+    fi
+    # displayName: 'Install and start servers {HDFS|minio}'
+
+    # Set up arguments for bootstrap.sh
+    bootstrap_args="--enable=verbose,static-tiledb";
+    if [[ "$TILEDB_HDFS" == "ON" ]]; then
+      bootstrap_args="${bootstrap_args} --enable-hdfs";
+    fi;
+    if [[ "$TILEDB_S3" == "ON" ]]; then
+      bootstrap_args="${bootstrap_args} --enable-s3";
+      # system libcurl on ubuntu 16.04 is linked against libssl1.0
+      # probably good to have one test doing full bootstrap anyway.
+      bootstrap_args="${bootstrap_args} --force-build-all-deps";
+    fi;
+    if [[ "$TILEDB_TBB" == "OFF" ]]; then
+      bootstrap_args="${bootstrap_args} --disable-tbb";
+    fi
+    if [[ "$TILEDB_DEBUG" == "ON" ]]; then
+      bootstrap_args="${bootstrap_args} --enable-debug";
+    fi
+    # displayName: 'Install dependencies'
+
+    # - bash: |
+    # Set up arguments for bootstrap.sh
+    bootstrap_args="--enable=verbose,static-tiledb";
```

Comment:
Nice catch, thanks. I initially had this broken down in to more sections for readability, which is why there are still commented out `- bash`/`displayName` sections (I'll restore those later). Had issues with that because variables don't propagate across task sections (unless you do [this trick](https://github.com/TileDB-Inc/TileDB/pull/1234/files/48ae15f538c9bcddb342c955039d5be42a3cea7d#diff-db24b7de0ecf80f45d04f82b47a56f12R42)), so simplified to one major block, but must have pasted twice in the process.